VTUL/iawa
# LIBTD-1871: Sort the sub-collections in ascending order by Name

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1871) (:star:)

# What does this Pull Request do? (:star:)
This PR sorts the list of sub-collections under the parent collection by the name in ascending order. In particular, what we really mean to sort by name is to have the resulting first page display "folder1, folder2, folder3, ......, folder10" but not "folder1, folder10, folder11, folder12......, folder18", then the second page "folder19, folder 2, folder3, ......". 

# What's the changes? (:star:)

* Set sort param as "system_create_dtsi asc" in Solr instead of "title_ssi asc" as the folders are uploaded in our expected order
* Make sure the returned solr docs in each page are ordered first by name then by the ending number within the name
* Append the customized collection_member_search_builder

# How should this be tested?

* Run as railsapps to upload a collection, e.g., bundle exec rails iawa:batch_import["Ms1998_022_Young/index.csv"] 
* See the list of sub-collections displayed in our expected order, e.g., in Ms1998_022_Young || Ms1998_022_Box3 collection

# Interested parties
Tag (@pmather) interested parties

(:star:) Required fields
